test/saucerunner.html

```diff
@@ -0,0 +1,32 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <!-- encoding must be set for mocha's special characters to render properly -->
+    <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
+    <script src="/test/vendor/lodash.min.js"></script>
+  </head>
+  <body>
+    <div id="deb"></div>
+    <div id="mocha"></div>
+    <script src="/node_modules/mocha/mocha.js"></script>
+
+    <script src="/node_modules/mocha-sauce/client.js"></script>
+
+    <script src="/test-compiled-for-web/ripple-for-web-tests.js"></script>
```

Comment:
It is exactly same, byte-to-byte. Standard build creates files with version number in name, so it will require introducing some form of templating into html.